### ISSUE TITLE
Make a dedicated recycling bin for coke cans

### ISSUE THREAD

**Issue by goatchurchprime** [FarsetLabs/somebody-should#17]
The bins for "mixed recycling" are way too small the the number of hollow space-wasting aluminium cans that would need to fit into them.  They're overflowing.  

Just because they're mixed when they go in the council bins doesn't mean it has to make sense to mix them in the bins in the hackspace.  They're massively bulkier and lighter than anything else and are much easier to handle separately.  

Also, you can use them as a resource, and flatten them to make an aluminium wall covering.  

Or run a Valentine's day make-your-own-coke-can-rose competition (judged by the mayor or some other dignitary who comes in at the end of the session) where the prize is a full can of coke.  
    https://www.pinterest.co.uk/explore/coke-can-crafts/

You can even serve popcorn at the event:
    https://www.instructables.com/id/Recycled-Coke-Can-Popcorn-Maker/

**Comment by clairebones** [comment]
People should ideally be crushing cans before putting them in the mixed recycling bins. 

If anyone finds the bins overflowing at any point, they are encouraged to empty them. This is considered the responsibility of every member and event organiser.

**Comment by andrewbolster** [comment]
There was an infamous ELi5 where @dsrkane demoed a coke-can-campfire 😂 

A fair amount of this comes down to people not emptying bins when there's an opportunity and just cramming things down. I try to remember to do at least one bin run to either the refuse, glass, or recycling bins (refuse and glass is directly behind farset, recycling is communal to the park and there's a map in the kitchen)
